components/Navigation/MobileNav/MobileNav.tsx

```diff
@@ -56,6 +57,15 @@ const MobileNavItem = ({
                         )}
                     </a>
                 </Link>
+            ) : link.isExternalLink ? (
```

Comment:
Looks good, I think if we add any new link types in the future we could refactor this into a few smaller link components so the ternary statements don't get too confusing